_posts/en/newsletters/2021-03-10-newsletter.md

```diff
@@ -43,7 +43,7 @@ projects.
       Town's proposal from last year described in [Newsletter
```

Comment:
`Towns's` is correct?

Reply:
Yes, `Towns's` is correct.  Good catch, thanks!

Reply:
Updated (thanks @azuchi!)